tls/s2n_cipher_suites.c

```diff
@@ -1074,6 +1074,33 @@ int s2n_cipher_suites_cleanup(void)
     return 0;
 }
 
+S2N_RESULT s2n_iana_to_cipher_suite(const uint8_t iana[], struct s2n_cipher_suite **cipher_suite)
+{
+    ENSURE_REF(cipher_suite);
+    *cipher_suite = NULL;
+    ENSURE_REF(iana);
+
+    int low = 0;
+    int top = s2n_array_len(s2n_all_cipher_suites) - 1;
```

Comment:
We could `size_t` instead of `int` here.

Reply:
It's dangerous to start removing signed-ness from this algorithm. For example, I _think_ 'top' can be unsigned, but if you make 'low' unsigned the algorithm will definitely break.